client/src/components/CommentEntry.jsx

```diff
@@ -32,17 +28,39 @@ const Content = styled.div`
 
 // components
 const CommentEntry = ({comment}) => {
+  const secondsToTime = (songTime) => {
+    songTime = Math.floor(songTime);
+    let result;
+    let hours = Math.floor(songTime / 3600);
+    let minutes = Math.floor(songTime - (hours * 3600)) / 60;
+    let seconds = songTime % 60;
+
+    if (hours > 0) {
+      result = Math.floor(hours);
+      result += `:${(minutes < 10 ? "0" + Math.floor(minutes) : minutes)}`;
+      result += `:${(seconds < 10 ? "0" + Math.floor(seconds) : seconds)}`;
+    } else {
+      result = `${Math.floor(minutes) + ':' + (seconds < 10 ? "0" + Math.floor(seconds) : seconds)}`;
+    }
+
+    return result;
+  }
+
   return (
     <CommentContainer>
-      <Space></Space> 
       <Avatar></Avatar> 
```

Comment:
Rememember to have this fit the styleguide (I think open/close tags must be on separate lines)